pom.xml

```diff
@@ -93,7 +93,7 @@
     <sonar.version>9.7.1.62043</sonar.version>
     <sonar.orchestrator.version>3.40.0.183</sonar.orchestrator.version>
     <sonar.api.version>9.11.0.290</sonar.api.version>
-    <sonar-analyzer-commons.version>2.1.0.1111</sonar-analyzer-commons.version>
+    <sonar-analyzer-commons.version>2.4.0.1285</sonar-analyzer-commons.version>
```

Comment:
I know this will be changed to a released version, but before merging this PR, can you update the version to the latest master build of analyzer commons (PR making `makePossessiveOrGreedy`overridable just got merged)?